csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -92,12 +98,12 @@ void mapAllocationDomain(
   for (auto* ref_id : ref_alloc_domain) {
     // skip when no ValGroup for ref_id to map.
     if (!val_graph.hasGroup(ref_id)) {
-      continue;
+      break;
```

Comment:
Comment could be updated here and below to reflect that we stop instead of skipping.